### PR TITLE
Fix wire randomization

### LINKED ISSUE #26015: Airlock wires generating incorrectly, or not displaying all wires.
No wire assigned to bolts or primary power.

### LINKED ISSUE #26017: Syndicate Bomb wire issue , unable to pry out the bomb core
## Can't pry out bomb core despite cutting "all" visible wires
Apparently only some wires show up. During a round it only gave me 3 wires, later when I tested it the bomb had 4. Despite cutting all wires it keeps giving you this message (see picture)

It used to have more wires for sure.
**How to reproduce:** Get a syndi bomb, cut all the wires and try to use a crowbar on it

### PR DESCRIPTION
**Pull request by octareenroon91**
Fixes wires.dm following #25325 .

:cl: 
fix: Hackable wires should work as intended again.
/:cl:

Fixes #26017
Fixes #26015

### CODE REVIEW

File: code/datums/wires/wires.dm [hunk 1]
```diff
@@ -74,9 +74,11 @@
 	"white",
 	"yellow"
 	)
+	
+	my_possible_colors = possible_colors.Copy()
```
Comment (Incoming5643) [inline code/datums/wires/wires.dm]: decent band name

File: code/datums/wires/wires.dm [hunk 2]
```diff
@@ -74,9 +74,11 @@
 	"white",
 	"yellow"
 	)
+	
+	var/list/my_possible_colors = possible_colors.Copy()
 
 	for(var/wire in shuffle(wires))
-		colors[pick_n_take(possible_colors.Copy())] = wire
+		colors[pick_n_take(my_possible_colors)] = wire
```
Comment (xxalpha) [inline code/datums/wires/wires.dm]: Are you sure this fix works, did you test it?

Comment (lzimann) [inline code/datums/wires/wires.dm]: This is not really changing anything, unless there's something odd going on with statics not being copied.

**Comment by WJohn** [comment]
Marking this as critical because there's a good chance you effectively cannot hack doors or autolathes or anything in certain rounds, which can have major game changing effects (accessing areas, printing weapons/ammo, controlling power etc).

**Comment by kevinz000** [comment]
Please test this..

**Comment by lzimann** [comment]
Well this actually works.

File: code/datums/wires/wires.dm (hunk 2 above)
Comment (xxalpha) [inline code/datums/wires/wires.dm]: It was making a new copy every time so it was always picking from a full list (making pick_n_take usage pointless).

Comment (lzimann) [inline code/datums/wires/wires.dm]: Oh, I see it, now it makes sense.

Comment (xxalpha) [inline code/datums/wires/wires.dm]: Yeah I didn't notice it at first either.

Comment (octareenroon91) [inline code/datums/wires/wires.dm]: Taken as a whole, it read like a single line in MATLAB or something, but it's catastrophic with the Copy() inside the for loop.

**Pull request merged**